<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateQrCodeDlg</class>
 <widget class="QDialog" name="GenerateQrCodeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GenerateQrCode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="queryGroupBox">
     <property name="title">
      <string>Query</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QDateEdit" name="startDateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="fromLabel">
        <property name="text">
         <string>from</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="queryPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>query</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="toLabel">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="endDateEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QGroupBox" name="generateGroupBox">
     <property name="title">
      <string>Generate</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QSpinBox" name="generateCountSpinBox">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="grainTypeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="generatePushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="printGroupBox">
     <property name="title">
      <string>Print</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="printPushButton">
        <property name="text">
         <string>print</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="selectAllPushButton">
        <property name="text">
         <string>selectAll</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="etcGroupBox">
     <property name="title">
      <string>Etc</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QPushButton" name="quitPushButton">
        <property name="text">
         <string>quit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="lockPushButton">
        <property name="text">
         <string>lock</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QTableView" name="generateQrCodeTableView">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>startDateEdit</tabstop>
  <tabstop>endDateEdit</tabstop>
  <tabstop>queryPushButton</tabstop>
  <tabstop>grainTypeComboBox</tabstop>
  <tabstop>generateCountSpinBox</tabstop>
  <tabstop>generatePushButton</tabstop>
  <tabstop>selectAllPushButton</tabstop>
  <tabstop>printPushButton</tabstop>
  <tabstop>lockPushButton</tabstop>
  <tabstop>quitPushButton</tabstop>
  <tabstop>generateQrCodeTableView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>quitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>GenerateQrCodeDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>738</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>686</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
